peps/pep-0765.rst

```diff
@@ -11,6 +11,7 @@ Post-History: `09-Nov-2024 <https://discuss.python.org/t/an-analysis-of-return-i
 Replaces: 601
 Resolution: https://discuss.python.org/t/71348/111
 
+.. canonical-doc:: https://docs.python.org/dev/reference/compound_stmts.html#finally-clause
```

Comment:
Using the reference: https://github.com/python/cpython/blob/8ad4646c675211dc1df0254a3160f50a18e4c6c3/Doc/reference/compound_stmts.rst?plain=1#L410C4-L410C12

```suggestion
.. canonical-doc:: :external+py3.14:ref:`finally`
```